### PR TITLE
ENH: add theme folder and minimal theme

### PR DESCRIPTION
**Pull request by mmcky**
This PR adds a minimal theme to support `HTML` and `PDF` for sphinx-quickstart.

It also includes the `first_notebook` from `sphinxcontrib-jupyter.minimal` as a `demo_notebook` that users can delete but gives them an idea of how `index` and `content` work together. 

- [x] @AakashGfude we should make this `demo_notebook` an option perhaps?

Results of the `minimal` theme are:

### index

image

### demo_notebook

image

### index

image

### demo_notebook

image

### CODE REVIEW

**Comment by mmcky** [comment]
fixes https://github.com/QuantEcon/jupinx/issues/50

**Comment by mmcky** [comment]
documented `warnings` issued https://github.com/QuantEcon/sphinxcontrib-jupyter/issues/295

**Comment by mmcky** [comment]
@AakashGfude if you could have a quick review of this and `approve` that would be great. I will release `0.2.1` probably on Tuesday/Wednesday.

File: jupinx/theme/minimal/templates/latex.tpl [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+% Insert citations in markdown as e.g.
+%    <cite data-cite="DevoretS2013">[DevoretS2013]</cite>
+% requires file references.bib in current directory (or the file set as "bib" in the latex_metadata)
+
+
+((* block bibliography *))
+((*- if nb.metadata.get("latex_metadata", {}).get("bib_include", ""): -*))
+% Add a bibliography block to the postdoc
+\bibliographystyle{plain}
+\bibliography{((( nb.metadata.get("latex_metadata", {}).get("bib", "quant-econ") )))}
```
Comment (mmcky) [inline jupinx/theme/minimal/templates/latex.tpl]: @AakashGfude this looks like it has some hard coding in it for the bibfile?

Comment (mmcky) [reply jupinx/theme/minimal/templates/latex.tpl]: @AakashGfude can we have the template use the bibfile specified in the `conf.py`? How do you get that variable?

File: jupinx/theme/minimal/templates/latex.tpl [hunk 2]
```diff
@@ -0,0 +1,304 @@
+((*- extends 'article.tplx' -*))
```
Comment (mmcky) [inline jupinx/theme/minimal/templates/latex.tpl:1]: @AakashGfude is this true?

Comment (AakashGfude) [reply jupinx/theme/minimal/templates/latex.tpl:1]: yes. it inherits from some of the base templates of nbconvert.

File: jupinx/theme/minimal/templates/latex.tpl (hunk 1 above)
Comment (AakashGfude) [reply jupinx/theme/minimal/templates/latex.tpl]: HI @mmcky, that `quant-econ` written in that line, is just a default value, when no bib file is specified. At present, we are including the bib file from the variable specified in `conf.py` itself.

**Comment by mmcky** [comment]
thanks @DrDrij and @AakashGfude. It will be great to have `jupinx` working out of the box for:

1. jupyter
1. html
1. pdf

**Note:** sphinxcontrib-jupyter is issuing a few uncessary warnings @AakashGfude which we should work through. But I will merge this and release a new version of `jupinx`

**Pull request merged**